plugins/woocommerce/src/Internal/Features/FeaturesController.php

```diff
@@ -262,6 +269,8 @@ private function get_feature_definitions() {
 	final public function init( LegacyProxy $proxy, PluginUtil $plugin_util ) {
 		$this->proxy       = $proxy;
 		$this->plugin_util = $plugin_util;
+
+		$this->plugins_excluded_from_compatibility_ui = $plugin_util->get_plugins_excluded_from_compatibility_ui();
```

Comment:
Instead of storing the list on init in the private instance variable, we could also always reference `$this->$plugin_util->get_plugins_excluded_from_compatibility_ui()` when we need the list. As far as I can tell there are only three lines in the FeaturesController where we need the list. Plus the method in PluginUtil is very simple -- it also just returns a private instance variable. Could it make sense to simplify things a bit and not keep a copy of the list in FeaturesController? Or are there any drawbacks that I'm not seeing? 🤔

Reply:
The method in PluginUtil is very simple _now_, but this could change in the future if we add some kind of filtering. On the other hand if for some reason we end up needing a more "dynamic" behavior (invoking the get plugins method each time), we can switch to doing that without breaking the public interface.